libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -292,7 +778,62 @@ genericBabbageFeatures pf =
             testU
               pf
               (trustMeP pf True $ referenceScriptTx pf)
-              (Right . utxoStEx2 $ pf)
+              (Right . utxoStEx2 $ pf),
```

Comment:
Just a syntactic observation that I've seen in this codebase many times, feel free to ignore it.

When there is only one composition it takes less characters to do this: 
`(Right $ utxoStEx2 pf)` vs `(Right . utxoStEx2 $ pf)`